srml/support/src/hash.rs

```diff
@@ -59,7 +59,7 @@ impl StorageHasher for Twox64Concat {
 	type Output = Vec<u8>;
 	fn hash(x: &[u8]) -> Vec<u8> {
 		twox_64(x)
-			.into_iter()
```

Comment:
why? the value (return value of `twox_64`) is being consumed and I find it nice that the code is implying ti by `into_`. Is there a performance implication here?

Reply:
It is equivalent to `iter()` because of arrays don't implement `IntoIterator [T; X]`, see https://github.com/rust-lang/rust/issues/25725. Thus, a readability thingy